src/sidebar/components/GroupList/test/GroupListItem-test.js

```diff
@@ -312,7 +315,10 @@ describe('GroupListItem', () => {
     });
     assert.equal(wrapper.find('MenuItem').first().prop('isDisabled'), true);
     const submenu = getSubmenu(wrapper);
-    assert.equal(submenu.exists('.GroupListItem__footer'), true);
+    assert.equal(
+      submenu.exists('[data-testid="unselectable-group-note"]'),
+      true
+    );
```

Comment:
I suggest to use `assert.isTrue` here. IIRC you get a more more readable error message than using `assert.equals` if the test fails.